model/k8s/metrics.yaml

```diff
@@ -126,3 +126,45 @@ groups:
     attributes:
       - ref: network.interface.name
       - ref: network.io.direction
+
+  # k8s.daemonset.* metrics
+  - id: metric.k8s.daemonset.current_scheduled_nodes
```

Comment:
For my own memory/context - These metrics don't have attributes because we expect the "Resource" they're associated with to be tied to the `k8s.daemonset` itself, allowing you to identify *which* daemonset you're talking about, right?

If so, could you document that in the `note` field?

Something like: 

```yaml
  note: |
     ... existing stuff...

     This metric SHOULD, at a minimum, be reported against a
     [`k8s.daemonset`](../resource/k8s.md#daemonset) resource.
```

Reply:
Right, for now we mention that in general for the whole set of these k8s metrics at https://github.com/open-telemetry/semantic-conventions/blob/v1.29.0/docs/system/k8s-metrics.md?plain=1#L15. Do you think we should do it per metric explicitly?

Reply:
How about proceeding with metrics definition and follow-up to add the specific resource relation?
Primary reason is that some metrics might not yet have resources defined so unblocking the metrics for now would really help with the progress.

Reply:
I agree we should not block on the tooling to get this PR in, but I think we should we add plain text in markdown saying which resources should be associated with these metrics and, if some of them are not defined yet, add a TODO comment ideally with a link to the issue to add these?

Reply:
Sounds good! Done in [aec1a00](https://github.com/open-telemetry/semantic-conventions/pull/1649/commits/aec1a007834cb5125085fec9b8e9fe5fa64b3d94)

Reply:
I had to rebase and resolve some conflicts. I squashed the commits as well.